one_fm/events/operations_post.py

```diff
@@ -0,0 +1,32 @@
+import frappe
+import datetime
+
+from frappe.utils import getdate, add_to_date, cstr
+
+
+def after_insert(doc, method=None):
+    try:
+        current_doc = frappe.get_doc("Operations Post", doc.name)   
```

Comment:
@NoBoneZ doc already contains the data no need to fetch it again.
You only created Post Schedule for 1 day, please reference existing post schedule code.